### ISSUE TITLE
File permissions!

### ISSUE THREAD

**Issue by GeorgeMcKinlay** [GeorgeMcKinlay/cheetOS#5]
Maybe a file permissions table for each drive? Or store it in the registry?

**Comment by LemonLake** [comment]
no

**Comment by LemonLake** [comment]
not sure if you have one already, but make a master file table and store their permissions there, or do something like inodes in unix

**Comment by GeorgeMcKinlay** [comment]
No, the file system is already written, there won't be a master file table.

**Comment by LemonLake** [comment]
then do an inode style thing

**Comment by LemonLake** [comment]
yes
